insights/specs/default.py

```diff
@@ -44,29 +44,45 @@ def get_owner(filename):
     return (name, group)
 
 
-def get_cmd_and_package_in_ps(broker, target_command):
+def _get_running_commands(broker, command):
     """
-    Search for command in ``ps auxww`` output and determine RPM providing binary
+    Search for command in ``ps auxcww`` output and determine RPM providing binary
 
     Arguments:
         broker(dict): Current state of specs collected by Insights
-        target_command(str): Command name to search for in ps output
+        command(str): Command name to search for in ps output
 
     Returns:
-        set: Set including all RPMs that provide the target command
+        list: List of the full command paths of the ``command``.
     """
-    ps = broker[DefaultSpecs.ps_auxww].content
+    ps = broker[PsAuxcww].search(COMMAND__contains=command)
     ctx = broker[HostContext]
-    results = set()
+    ret = set()
     for p in ps:
-        p_splits = p.split(None, 10)
-        cmd = p_splits[10].split()[0] if len(p_splits) == 11 else ''
-        which = ctx.shell_out("which {0}".format(cmd)) if target_command in os.path.basename(cmd) else None
-        resolved = ctx.shell_out("readlink -e {0}".format(which[0])) if which else None
-        pkg = ctx.shell_out("/bin/rpm -qf {0}".format(resolved[0])) if resolved else None
-        if cmd and pkg is not None:
-            results.add("{0} {1}".format(cmd, pkg[0]))
-    return results
+        cmd = p['COMMAND_NAME']
+        if cmd != command:
+            continue
+        which = ctx.shell_out("/usr/bin/which {0}".format(cmd))
+        ret.add(which[0]) if which else None
+    return sorted(ret)
```

Comment:
@xiangce since a `set` doesn't have an order, should this be `sorted(list(set))`?

Reply:
@bfahr - here it returns a `sorted(set)` but not the original `set`. I think the `sorted(set)` is equivalent with `sorted(list(set))`, both of them returned a new sorted list:
- https://docs.python.org/3/library/functions.html#sorted